### ISSUE TITLE
Choose the fields available in new contacts

### ISSUE THREAD

**Issue by livier** [nextcloud/contacts#3615]
### Is your feature request related to a problem? Please describe.

Not a big problem : I often use "notes" fields in (new) contacts. Each time i need it I have to click 3 times for "ajouter plus d'information" (here in french) > "plus de champs" > scroll > "notes". 
Should it be possible to setup something anywhere to choose the fields I want to have directly available at each new contact I create. 

### Describe the solution you'd like

A place in the setups for user or admin

### Describe alternatives you've considered

Different possibilities would be helpfull as ...
- Une "case à cocher" to say "always have this field in new contacts, when we add it for one off them
- some occ command 
- some line to right in the config file

**Comment by punkyard** [comment]
hi
exact same hopes for us

**Comment by ChristophWurst** [comment]
@punkyard to express your interest in a feature add your :+1: (like you already have). Commenting is not necessary. It even causes an avoidable notification for maintainers.

**Comment by punkyard** [comment]
Hi 
in iOS devices, the contact name is not shown
all Apple devices need the full name fields
could you show the full name fields by default?
plus, the 'short name' field brings a bug the the company name when syncing with iOS 
thank you :)

**Comment by Peque** [comment]
Duplicate of https://github.com/nextcloud/contacts/issues/1951 ?

There may be multiple issues about the same feature, actually (from older to newer): #1951, #3385, #3615, #4537

**Comment by SebastianKrupinski** [comment]
@Peque yes, thank you, will close this as a duplicate, and transfer the votes.
